MapboxNavigation/UserCourseView.swift

```diff
@@ -113,5 +113,143 @@ class UserPuckCourseView: UIView, UserCourseView {
             puckArrow?.setAffineTransform(CGAffineTransform.identity.rotated(by: -CGFloat(angle.toRadians())))
         }
     }
+}
+
+class NewUserPuckCourseView: UIView, UserCourseView {
```

Comment:
The idea is to eventually replace UserPuckCourseView with NewUserPuckCourseView, right?

Reply:
Yes